cluster/log-dump.sh

```diff
@@ -128,6 +129,9 @@ function save-logs() {
       case "${KUBERNETES_PROVIDER}" in
         gce|gke|kubemark)
           files="${files} ${gce_logfiles}"
+          if [[ "${KUBERNETES_PROVIDER}" -eq "kubemark" && "${ENABLE_HOLLOW_NODE_LOGS:-}" -eq "true" ]]; then
```

Comment:
Sorry for the blooper. This is fixed in PR https://github.com/kubernetes/kubernetes/pull/41800.